packages/paste-libraries/styling/src/css-function/css.ts

```diff
@@ -0,0 +1,224 @@
+// Very lightly TS Version of https://github.com/styled-system/styled-system/blob/master/packages/css/src/index.js
```

Comment:
Mainly copy pasta from styled-system. Simplified the get method a little, renames some arguments in it